wasm/generateInterface.js

```diff
@@ -199,7 +199,64 @@ function generateManifest (interfaceManifest, opts) {
     let spOffset = 0
     let numOfLocals = 0
     let lastOffset
+
     let call = `(call $${op.name}`
+
+    // handle some EEI pointer special cases
+    if (opcode === 'DIFFICULTY') {
+
+      op.input.forEach((input) => {
+        if (input === 'pointer') {
+          // a pointer input is more like an output
```

Comment:
Why not have two types: `ipointer` and `opointer` or (`inpointer`/`outpointer`)?